manifests/install.pp

```diff
@@ -3,4 +3,10 @@
   package { $katello::package_names:
     ensure => installed,
   }
+
+  if ($katello::enable_ostree) {
+    package { [$katello::params::rubygem_katello_ostree]:
```

Comment:
Don't reference params outside of the class parameter definitions -- also I don't see 'rubygem_katello_ostree' defined anywhere.